CMakeLists.txt

```diff
@@ -61,13 +62,14 @@ endif()
 
 # Increment this for each rebuild (i.e. new kernel) using the same upstream ver
 set(CPACK_DEBIAN_PACKAGE_RELEASE "1")
+set(GREENZIE_SUFFIX "greenzie-focal")
```

Comment:
This is a public repo, so we should try to keep anything "Greenzie" specific out of it, IMO. Maybe make this another variable to be passed in?

Reply:
Updated logic, made it generic and optional, if variable `VER_SUFFIX` exist add it, if not defaults to empty string